packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -1,32 +1,40 @@
 <h6 class="preview__label">English banner</h6>
 <section class="ds-c-usa-banner" aria-label="Official government website">
   <header class="ds-c-usa-banner__header">
-    <div class="ds-c-usa-banner__inner">
-      <p class="ds-c-usa-banner__header-text">
-        <svg
-          width="16"
-          height="11"
-          class="ds-c-usa-banner__header-flag"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <use href="{{root}}/images/usa-banner-flag.svg#svg"></use>
-        </svg>
-      </p>
-      <p class="ds-c-usa-banner__header-text">
-        <span>An official website of the United States government</span>
-        <span class="ds-c-usa-banner__header-action" aria-hidden="true">Here’s how you know</span>
-        <button class="ds-c-usa-banner__button" aria-expanded="true" aria-controls="gov-banner">
-          <span class="ds-c-usa-banner__button-text">Here’s how you know</span>
-        </button>
-      </p>
-    </div>
+    <p class="ds-c-usa-banner__header-text">
+      <svg
+        role="img"
+        width="16"
+        height="11"
+        xmlns="http://www.w3.org/2000/svg"
+        class="ds-c-usa-banner__header-flag"
+        aria-labelledby="banner-flag-title-1 banner-flag-description-1"
+        focusable="false"
+      >
+        <title id="banner-flag-title-1">Flag</title>
+        <desc id="banner-flag-description-1">U.S. flag</desc>
+        <use href="{{root}}/images/usa-banner-flag.svg#svg"></use>
```

Comment:
Noticed that on IE 11, the svg `<use... />` command is picking up the image content on this banner html example. Have confirmed that it is the same for `button with Icon` component where it works on other browsers except for IE11.

Reply:
nice job testing, we can file a separate ticket for that issue